alexa.utterances

```diff
@@ -648,3 +648,25 @@ WhatNewShows what shows do you have
 WhatNewShows what shows i have
 WhatNewShows what shows we have
 WhatNewShows what shows you have
+WatchPVRBroadcast watch the program called {Broadcast}
+WatchPVRBroadcast play the program called {Broadcast}
+WatchPVRBroadcast find the broadcast called {Broadcast}
+WatchPVRBroadcast watch the broadcast called {Broadcast}
+WatchPVRBroadcast play the broadcast called {Broadcast}
+WatchPVRBroadcast look for the program called {Broadcast}
+WatchPVRBroadcast look for the broadcast called {Broadcast}
+WatchPVRBroadcast look for {Broadcast}
+WatchPVRBroadcast where is the program called {Broadcast}
+WatchPVRBroadcast where is the broadcast called {Broadcast}
+WatchPVRBroadcast where is {Broadcast}
+WatchPVRBroadcast find the program called {Broadcast}
+WatchPVRBroadcast find the broadcast called {Broadcast}
+WatchPVRBroadcast find {Broadcast}
+WatchPVRBroadcast watch something called {Broadcast}
+WatchPVRBroadcast watch anything called {Broadcast}
+WatchPVRBroadcast jump to {Broadcast}
+WatchPVRBroadcast change channel to {Broadcast}
+WatchPVRBroadcast change channel to watch {Broadcast}
+WatchPVRBroadcast change channel to play {Broadcast}
+WatchPVRChannel watch {Channel}
+WatchPVRChannel play {Channel}
```

Comment:
Watch and Play need a modifier, at least for now.

I want to make it so the skill can cope with "play <foo>", but at least currently, "play <foo>" is mapped to music artists.

Ideally, "play <foo>" would attempt to search through all available media and do smart things, but it's not there yet.